portal/views/home.py

```diff
@@ -54,8 +52,9 @@
 from portal import app_settings, emailMessages
 from ratelimit.decorators import ratelimit
 from portal.forms.home import ContactForm
+from portal.captcha import check_recaptcha
 
-recaptcha_client = RecaptchaClient(app_settings.RECAPTCHA_PRIVATE_KEY, app_settings.RECAPTCHA_PUBLIC_KEY)
+from portal import app_settings
```

Comment:
Duplicate import? It looks like `portal.app_settings` are already imported on line 52 :stuck_out_tongue: